frontend/components/CourseEvaluation/SectionTabs.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { useSWRAuth } from '@/components/hooks/useSWRAuth';
+import {
+  API_ENDPOINT,
+  CourseEvaluationListEntry,
+  DEFAULT_COURSE_EVALUATION_LIST_ENTRY,
+} from 'utils/api';
+import CustomTheme from '../utils/CustomTheme';
+
+import Overview from './Overview';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+type Props = {
+  id: string;
+};
+
+function SectionTabs({ id }: Props) {
+  const { response, isLoading, error } = useSWRAuth(API_ENDPOINT.COURSE_EVALUATION.LIST);
+
+  const courseEvaluationListEntries = ((response?.data as unknown) ||
+    []) as CourseEvaluationListEntry[];
+
+  let evaluation: CourseEvaluationListEntry = DEFAULT_COURSE_EVALUATION_LIST_ENTRY;
+
+  //looping through all evaluations to find the one that is being asked for
+  courseEvaluationListEntries.forEach(function (value) {
+    if (value.id == id) {
+      evaluation = value;
+      return;
+    }
+  });
```

Comment:
this is inefficient... there is a better way.

currently you are using the endpoint called `/api/v1/course-evaluations/`... what you should be using instead is `/api/v1/course-evaluations/${courseEvaluationId}`. This will contain more detail about this specific courseEvaluation.


I would recommend to create a new entry in the `utils/api.ts`

```
COURSE_EVALUATION: {
    LIST: '/api/v1/course-evaluations/',
    DETAIL: (courseEvaluationId) => `/api/v1/course-evaluations/${courseEvaluationId}`
  }
```

and use it like  (something like this)
```
const { response, isLoading, error } = useSWRAuth( courseEvaluationId ? API_ENDPOINT.COURSE_EVALUATION.DETAIL(courseEvaluationId) : "" );
```